src/pykeen/ablation/__init__.py

```diff
@@ -22,8 +22,6 @@
         },
         directory='~/Desktop/simple_ablation_study',
```

Comment:
@mali-git what should happen to the documentation stub here?

Reply:
This can be removed (done in e93807d9810b0543abcf7227b9305e45c0043cde).